{{:Team:SCU-China/NavBar}}
<html>
      <div class="container content project">
        <div class="aqua-bk padding-bottom">
          <div class="content-block">
            <h1 class="center">Enhancement</h1>
            <p>E. coli odor itself is not that pleasant as well. We use RED to knockout the insole production gene tnaA (tryptophanase) to reduce the odor itself. Insole in shoes is not a wonderful home for microorganisms to grow. We use vitreoscilla hemoglobin (existing part K1321200) to enhance the grow ability of two E. coli strains and get the quantitative measurement result of the Biobrick.</p>
          </div>
        </div>
        <div class="padding-bottom">
          <div class="content-block">
            <h1 id="eOverview">Overview</h1>
            <div class="hr-black"></div>
            <p>Apart from the foot odor, the smell of E. coli itself is also a problem remained to be solved, which is imperative for gaining acceptability for our product. The smell is believed that mainly comes from the derivatives of indole which mostly originate from indole. Thus shutting down the production of indole becomes the key task. The direct way of indole formation in E. coli is the process that turns a tryptophan into indole by tryptophanase. As a result, turning off this process by knocking out tryptophanase will reduce the source of indole as well as the smell. We decide to use RED system to knockout the gene coding tryptophanase: tnaA. [1]</p>
            <p>Insole in shoes is not a wonderful home for microorganisms to grow. The oxygen and nutrition are both limited. We use vitreoscilla hemoglobin (existing part K1321200) to enhance the grow ability of two E. coli strains and get the quantitative measurement result of the Biobrick. And it’s the “Biobrick character improvement” in our project this year. We prove that it can be expressed in E.coli and get a lot of quantitative measurement result.</p>
            <p>[1] 2014 iGEM Team:Paris_Saclay, “Remove the bad smell of E.coli” <a href="http://2014.igem.org/Team:Paris_Saclay/Project/Odor-free_ecoli">http://2014.igem.org/Team:Paris_Saclay/Project/Odor-free_ecoli</a>， 2016/10/4</p>
          </div>

          <div class="content-block">
            <h1 id="eK">Knockout</h1>
            <div class="hr-black"></div>
            <h2>Tryptophanase-Knock-out Group</h2>
            <p><strong>Goal:</strong> Indole and its derivatives are believed to be the important source of the E. coli odor. Our goal it to knock out tnaA, the essential gene for indole production in Escherichia coli for reducing odor of E. coli itself by RED recombination system. HPLC is introduced for testing the result.</p>
            <p><strong>Achievement:</strong> tnaA is successfully knocked out in the JM109 string of E. coli. Result from HPLC shows that the concentration of indole and methylindole are both evidently declined in the mutant strain, with the improvement of the smell and drug resistance declined.</p>
            <p><strong>Part:</strong> BBa_K1919400</p>
            <h2>Background:</h2>
            <p>Apart from the foot odor, the smell of E. coli itself is also a problem remained to be solved, which is imperative for gaining acceptability for our product. The smell mainly comes from the derivatives of indole which mostly originate from indole. Thus shutting down the production of indole becomes the key task. The direct way of indole formation in E. coli is the process that turns a tryptophan into an indole molecule by tryptophanase. As a result, turning off this process by knocking out tryptophanase will reduce the source of indole as well as the smell.</p>
            <h2>Deisgn:</h2>
            <p>Our biobrick is the homologous substitution fragment for RED recombination, which contains a chloramphenicol-resistant gene with a homology arm on each side. It is for replacing the tryptophanase gene. </p>
            <p>RED recombination system is selected for the knocking out because it has already been a mature method of knocking out genes in the bacteria genome.</p>
            <p>The steps are as follows:</p>
            <p>1. Primers designing.</p>
            <p>2. Amplify the drug-resistant gene with homology arms.</p>
            <p>PCR. Using primer A as primer and pKD3 plasmid as template to build the Biobrick of a chloramphenicol-resistant gene with a homology arm on each side. Gel extraction was performed to purify DNA fragment.</p>
            <p>3. Transforming PKD46 plasmid into the competent cells</p>
            <p>[Notice] (1) Screening with Amp for positive colonies. (2) All the cultures are performed at 30 degrees centigrade.</p>
            <p>4. Formation of electro-transforming competent bacterial cells</p>
            <p>Vaccinate the bacteria from step 3 at 1% concentration and culture for 1 hour at 30 degrees centigrade. Then add in L-arabinose (2% as final concentration) to induce for at least 90 minutes, forming electro-transforming competent bacterial cells. Add 10% amount of glycerol and make sure the total volume is under 100ml.</p>
            <p>5 Electro-transformation</p>
            <p>Chill down the electro-transformation cup in advance, and thaw the competent cells on the ice. Then add in the regenerant of gel extraction in step 2, mix up, move onto the ice in electro-transformation cup, stay for 10 minutes, and then start the electric shock.</p>
            <p>[Notice] 1. The amount of the regenerant added into the cells depends on its concentration. Generally we add 2 to 6 microlitres, the higher the concentration is, the less we add. 2. Act rapidly when perform electrotransformation.</p>
            <p>6. Screening for strains.</p>
            <p>Spread bacterial fluid on chloramphenicol plate at 30 degrees centigrade, and pick single colonies into liquid culture medium, shake for 8 hours. Then adjust the temperature to 42 degrees centigrade. Continue to culture for 6 hours, and then perform PCR to figure out the possible aim strain, and use glycerol to conserve. Figure out the aim strain by sequencing.</p>
            <p>7. Analyzing the concentration of indole and methylindole in the aim strain with HPLC</p>
            <h2>Results:</h2>
            <div class="figure">
              <img src="http://parts.igem.org/wiki/images/9/90/T--SCU-China--knockoutimage1.png">
              <p>Diagram1. PCR screening of the aim strain</p>
              <img src="http://parts.igem.org/wiki/images/d/da/T--SCU-China--knockoutimage2.png">
              <p>Diagram2.centration curve of indole（mutant：blue line& wild type：yellow line）</p>
              <img src="http://parts.igem.org/wiki/images/5/55/T--SCU-China--knockoutimage3.png">
              <p>Diagram3.bacteria growth curve （mutant：yellow line & wild type：blue line）</p>
            </div>
            <h2>Discussion:</h2>
            <p>RED recombination system is capable of achieving effective knocking out of tryptophanase gene, and the concentration of indole and methylindole are both evidently declined after the knocking, leading to the improvement of the smell. Yet meanwhile the quorum sensing among the bacteria is weakened as a result of the lack of indole as indole works as an important signal molecule between different individuals of the bacteria, so the drug resistance of the mutant strain has declined.</p>
          </div>
        </div>
      </div><!-- /.container -->

<script>
  $(document).ready(function() {
      $(".active").removeClass("active");
      $(".list2").addClass("active");
  });
</script>
  </body>
</html>
