{{:Team:SCU-China/NavBar}}
<html>
  <body>
      <div class="container content project">
        <div class="aqua-bk padding-bottom">
          <div class="content-block">
            <h1 class="center">Rihanna</h1>
            <p>You can stand on my umbrella to ANTI microorganisms.</p>
            <p>Antimicrobial peptide CecropinXJ is synthesizing in “Rihanna” anytime. When your foot is on the insole, the temperature of your foot induces the temperature sensitive promoter to drive the expression of self-kill gene hokD to lyse the “Rihanna” to release CecropinXJ.</p>
            <p>Visit Project “Rihanna” for more.</p>
          </div>
        </div>
        <div class="padding-bottom">
          <div class="content-block">
            <h1 id="rOverview">Overview</h1>
            <div class="hr-black"></div>
            <p>The best way to control foot odor and tinea pedis is to control the microorganisms on our feet. It’s well known that fungus can cause tinea pedis: most commonly caused by Trichophyton rubrum, a fungi species originated from Southeast Asia. But in different areas, the species of fungi causing infection varies sometimes. [1]. For foot odor, researches have shown that gram positive microorganisms contribute to the foot odor (Figure 1) [2]. </p>
            <div class="figure">
              <img src="http://igem.org/wiki/images/7/74/T--SCU-China--p-r-o-f1.png">
              <p>Figure 1</p>
            </div>
            <p>So we decided to use engineered bacteria to secret chemical components to inhibit them. Antibiotics are not a wise choice because the abuse of them are very serious now and it’s useless for fungus inhibition. As a result, we decide to utilize antimicrobial peptide (AMP) to deal with the problem. It’s important to find a antimicrobial peptide which has a wide antimicrobial spectrum towards gram positive bacteria and fungus and can be expressed in E. coli successfully.  At last, we find CecropinXJ. It can be expressed in E. coli in fusion protein form and has a wide antimicrobial spectrum (Figure 2) [3], which is suitable for our goal.</p>
            <div class="figure">
              <img src="http://igem.org/wiki/images/8/8f/T--SCU-China--p-r-o-f21.png">
              <img src="http://igem.org/wiki/images/8/8b/T--SCU-China--p-r-o-f22.png">
              <p>Figure 2</p>
            </div>
            <p>But another problem is that, how to release the antimicrobial peptide? The secretion system in E. coli is not easy for releasing of protein synthesized in E.coli even with signal peptide. As a result, we decide to lyse the cell directly to release the content by temperature inducement.</p>
            <p>The Rihanna system works as follows.</p>
            <div class="figure">
              <img src="http://igem.org/wiki/images/f/f3/T--SCU-China--i-3c.gif">
              <img src="http://igem.org/wiki/images/8/87/T--SCU-China--i-3h.gif">
              <p>Figure 3</p>
            </div>
            <p>[1] Michael J Wells et al. "Tinea Pedis" <a href="http://emedicine.medscape.com/article/1091684-overview">http://emedicine.medscape.com/article/1091684-overview</a>. 2016/10/4.</p>
            <p>[2] James, A. Gordon, D. Cox, and Kathryn Worrall ‡. "Microbiological and biochemical origins of human foot malodour †." Flavour & Fragrance Journal 28.4(2013):231–237.</p>
            <p>[3] Xia, Lijie, et al. "Expression and characterization of cecropinXJ, a bioactive antimicrobial peptide from Bombyx mori (Bombycidae, Lepidoptera) in Escherichia coli." Experimental & Therapeutic Medicine 5.6(2013):1745-1751.</p>
          </div>

          <div class="content-block">
            <h1 id="rC">CecropinXJ</h1>
            <div class="hr-black"></div>
            <h2>Overview</h2>
            <p><strong>Goal:</strong> To make AMP CecropinXJ being expressed by E.coli strain Rihanna.</p>
            <p><strong>Achievements:</strong> Successful prokaryotic expression of CecropinXJ was indicated by the results of SDS-PAGE and Western blot. The results of inhibition zone assay exhibits effective antimicrobial activity of this peptide towards 病原菌XXX, XXX, XXX etc. (还没做出来，先这样写着)</p>
            <p><strong>Parts:</strong> BBa_K1919000, BBa_K1919001, BBa_K1919002, BBa_K1919003</p>

            <h2>Introduction</h2>
            <p>Antimicrobial peptides (AMPs) are a group of peptides that play roles in the innate immune system to protect the host from invading pathogens [1]. AMPs have minimal toxicity and low sensitivity effects to the host [2], which means antimicrobial peptides have the potential to be used to replace antibiotics in the future. Thus, the detrimental effects of antibiotics overuse will be released.</p>
            <p>Cecropins, a group of small AMPs mainly found in the hemolymph of insects, consist of 31‑39 amino acid residues and have a broad spectrum, high heat stability and potent bacteriostatic activity [3-5]. CecropinXJ (Part BBa_K1919000) is a member of the Cecropin family, which was first cloned from the larvae of the Xinjiang silkworm (Bombyx mori). Previous researches have determined the complete amino acid sequence of this molecule [6]. It has been demonstrated that CecropinXJ could be expressed in eukaryotic expression system such as Pichia pastoris [7] or prokaryotic expression system such as E.coli [8]. What’s more, CecropinXJ exhibited to have various activities such as antibacterial activity against both Gram‑positive and Gram-negative bacteria, as well as antifungal activity [8]. These characteristics indicate that CecropinXJ is an ideal antimicrobial substance to be used to treat foot diseases caused by microbes.</p>
            <p>In recent years, studies concerning the expression of anti­microbial peptides have mainly focused on the use of fusion partners [9]. For example, Thioredoxin, a heat-stable and low molecular weight soluble protein in the prokaryotic cytoplasm, has been shown to display chaperone‑like activity [10]. Fusion proteins of antimicrobial peptides generated in E. coli reduce the toxic effect of antimicrobial peptides to the host cells and protect the small antimicrobial peptides from proteolytic degradation.</p>

            <h2>Design</h2>
            <p>Based on the various antimicrobial activities performed by CecropinXJ, we aimed to use hire this peptide as the main antimicrobial substrate in our insole. In order to realize it, CecropinXJ was expressed in our E.coli strain Rihanna, which is the main attacker grow in our insole.</p>
            <p>We employed 3 different promotors to construct 3 individual expression plasmids respectively. These promotors are: T7 promotors (inductive), promotor J23105 (constitutive), promotor J23106 (constitutive). Downstream of promotors, the coding sequences are consistent: TrxA tag/6 x His tag/Thrombin site/S-tag/enterokinase site/CecropinXJ CDS. The 5-tag system exists in commercial plasmid pET32a(+), which could be easily constructed through PCR. The final plasmid products are BBa_K1919001, BBa_K1919002 and BBa_K1919003</p>
            <p>After plasmids construction, our target proteins are expressed using E.coli BL21(DE3) under appropriate conditions. Protein expression level was determined through SDS-PAGE and Western Blot (Anti-6 x His tag). Finally, the antimicrobial activities are determined through inhibition zone assay. We aimed to test the inhibition level on the growth of Staphylococcus aureus (Foot odor pathogen), Bacillus subtilis (Foot odor pathogen), Klebsiella pnuemoniae (Quality control strain), Escherichia coli (Quality control strain), Enterococcus faecalis (Quality control strain), Microsporum canis (Tinea Pedis pathogen) and Trichophyton rubrum (Tinea Pedis pathogen). Due to lab condition limitation, we performed S. aureus and E. coli inhibition tests in our lab, while the tests on other microbes were performed by our collaborator TMMU_China.</p>

            <h2>Results</h2>
            <p>Considering the fact that an antimicrobial peptide expressed in bacteria may be cytotoxic to the host or subjected to degradation by host-derived peptidases [11,12], we used the recombinant expression system to overcome the potential problems ——fusing the DNA coding sequence of CecropinXJ with the sequence of a bacterial thioredoxin gene exists in the pET32a(+) expression system [3,4]. To construct recombinant pET32a-cecropinXJ expression vector (Fig.1B), we used artificially synthetic CecropinXJ synthesized by Tsingke Ltd. (Fig.1A). Subsequently, CecropinXJ and pET32a plasmid were subjected to enzymatic digestion with EcoRI and XhoI, and ligated using T4 DNA ligase.</p>
            <div class="figure">
              <img src="http://igem.org/wiki/images/7/7b/T--SCU-China--p-r-c-f11.png">
              <img src="http://igem.org/wiki/images/7/7e/T--SCU-China--p-r-c-f12.png">
              <p>Fig.1 Schematic structure of  CecropinXJ in pSB1C3(the upper one) and CecropinXJ in pET32a (the lower one).</p>
            </div>
            <p>After that, we selected positive clones which were resistant to ampicillin on LB plate to confirm the plasmid through PCR (Fig.2) and DNA sequencing. The verified recombinant plasmid was transformed into the E. coli strain BL21(DE3) pLYsS competent cells [8], which encodes a chromosomal T7 RNA polymerase under the control of a tac promoter. </p>
            <div class="figure">
              <img src="http://igem.org/wiki/images/4/4b/T--SCU-China--p-r-c-f2.png">
              <p>Fig.2 The result of colony PCR confirmed the successful construction of CecropinXJ expression plasmid.</p>
            </div>
            <p>When the optical density at OD600nm of the culture reached 0.6-0.8, we added 0.8mM IPTG to induce the cells, by which time the tac promoter was activated and drove expression of pET32a-CecropinXJ. After 5 hours induction at 37℃, the expression level of recombinant CecropinXJ was detected through SDS-PAGE and western blot analysis. In the result of SDS-PAGE (Fig.3), an obvious band at the size of 25 kDa compared with control was observed, which was as expected. The result of western blot analysis provided subsequent confirmation of expression (Fig.4). </p>
            <div class="figure">
              <img src="http://igem.org/wiki/images/6/6c/T--SCU-China--p-r-c-f3.png">
              <p>Fig.3  Expression of pET32a-cecropinXJ fusion protein analyzed by SDS‑PAGE.</p>
            </div>
            <p>However, since the western blot analysis was used on the whole cell proteins, some non-specific bindings were also obtained. Nonetheless we can still make a clear determination through the comparison between the induced cell and the control based on the high quantity of the expression product. Besides, in the lane of the pET32a factor without CecropinXJ gene, we also obtained a clear expression band at the size of about 20kDa, which is the protein product of the 5-tag system encoded by pET32a expression system itself. In addition, the purification of recombinant protein by Ni-NTA was performed under the assistance of TMMU_China.</p>
            <div class="figure">
              <img src="http://igem.org/wiki/images/e/e2/T--SCU-China--p-r-c-f4.png">
              <p>Fig.4 Western Blotting result of cecropinXJ expression</p>
            </div>
            <p>According to the results, we can see that the E.coli recombinant expression system is a good way to produce AMPs because of its easy culture, fast growth and larger quantities than those purified from their natural sources. What’s more, this system costs less money compared with chemical synthesis.</p>
            <p>Besides, another two vectors which express cecropinXJ continuously downstream of two different constitutive promoters, J23105(Fig. 5A) and J23106(Fig. 5B) were also constructed.</p>
            <div class="figure">
              <img src="http://igem.org/wiki/images/4/45/T--SCU-China--p-r-c-f51.png">
              <img src="http://igem.org/wiki/images/0/0b/T--SCU-China--p-r-c-f52.png">
              <p>Fig.5 Schematic structure of  CecropinXJ with J23105 (the upper one) and CecropinXJ with J23106(the lower one)</p>
            </div>
            <p>The combination of cecropinXJ and constitutive promoters were determined by PCR.( Fig. 6)</p>
            <div class="figure">
              <img src="http://igem.org/wiki/images/3/3f/T--SCU-China--p-r-c-f61.png">
              <img src="http://igem.org/wiki/images/5/5e/T--SCU-China--p-r-c-f62.png">
              <p>Fig.6 The result of colony PCR confirmed the successful combination of CecropinXJ and constitutive promoters J23105(the upper one) /J23106(the lower one)</p>
            </div>
            <p>To detect the antimicrobial activity of the recombinant CecropinXJ, we did the inhibition zone assay towards Staphylococcus aureus (as well as Bacillus subtilis, Klebsiella pnuemoniae, Escherichia coli, Enterococcus faecalis, Microsporum canis and Trichophyton rubrum with the assistance of TMMU_China) and detected the growth situation of them after recombinant CecropinXJ was added into the bacterial culture solution. To obtain recombinant CecropinXJ solution, 10ml culture was centrifuged at 8,000 x g for 5 min after induction. The pellet was resuspended in 10 ml PBS and placed in an ice bath for ultrasonic lysis (200 W, 5 sec, 5 sec). The lysate was centrifuged at 10,000 x g for 5 min and supernatant was collected for further work.
</p>
            <p>To simulate the real environment of our insole, we used the raw ultrasonic cell lysate for bacterial inhibition test. Staphylococcus aureus were grown in LB at 37℃. A dilution of Staphylococcus aureus (20µl; OD600=0.5) was taken and added to 1.5ml, 1.0ml and 0.5ml ultrasonic lysate containing recombinant CecropinXJ respectively and then 0.5ml, 1.0ml and 1.5ml of fresh LB was added accordingly. After incubation at 37℃ for 0.5h, 1.0h and 1.5h, the absorbance of culture at 600nm (OD600) was detected respectively using spectrophotometer(figure xx/table xx).</p>
            <p>For the inhibition zone assay, we used the filter paper method. The filter paper was soaked in the raw ultrasonic lysate for 10min and put on the medium after plate coating of Staphylococcus aureus. The antibacterial efficacy and inhibition zone was shown in picture xxx. </p>
            <p>Our results show that the recombinant CecropinXJ has potent antimicrobial activity. Earlier studies had indicated that CecropinXJ shares a similar structure with ABP-CM4, which has the ability to form specific amphipathic α-helices which allows targeting of nonpolar lipid cell membranes. Upon membrane targeting, the helices form ion-permeable channels, subsequently resulting in cell depolarization, irreversible cytolysis and cell death [15-17].</p>
            <p>[1] Boman HG: Peptide antibiotics and their role in innate immunity. Annu Rev Immunol 13: 61-92, 1995.</p>
            <p>[2] Devine DA and Hancock RE: Cationic peptides: distribution and mechanisms of resistance. Curr Pharm Des 8: 703-714, 2002.</p>
            <p>[3] Boman HG, Wade D, Boman IA, Wåhlin B and Merrifield RB: Antibacterial and antimalarial properties of peptides that are cecropin-melittin hybrids. FEBS Lett 259: 103-106, 1989.</p>
            <p>[4] Moore AJ, Devine DA and Bibby MC: Preliminary experimental anticancer activity of cecropins. Pept Res 7: 265-269, 1994.</p>
            <p>[5] Hancock RE and Lehrer R: Cationic peptides: a new source of antibiotics. Trends Biotechnol 16: 82-88, 1998.</p>
            <p>[6] Li JY, Zhang FC and Ma ZH: Prokaryotic expression of cecropin gene isolated from the silk worm Bombyx mori Xinjiang race and antibacterial activity of fusion cecropin. Acta Entomol Sin 47: 407-411, 2004 (In Chinese).</p>
            <p>[7] Tang X, Wang H, Kelaimu R, Mao XF and Liu ZY: Molecular cloning, expression of cecropin-XJ gene from silkworm and antibacterial activity in Pichia pastoris. Biotechnology 21: 26-31, 2011 (In Chinese).</p>
            <p>[8] Xia L, Zhang F, Liu Z, Ma J and Yang J: Expression and characterization of cecropinXJ, a bioactive antimicrobial peptide from Bombyx mori (Bombycidae, Lepidoptera) in Escherichia coli. Experimental and Therapeutic Medicine 5: 1745-1751, 2013.</p>
            <p>[9] Zorko M and Jerala R: Production of recombinant antimicrobial peptides in bacteria. Methods Mol Biol 618: 61-76, 2010.</p>
            <p>[10] LaVallie ER, Diblasio EA, Kovacic S, Grant KL, Schendel PF and McCoy JM: A thioredoxin gene fusion expression system that circumvents inclusion body formation in E. coli cytoplasm. Biotechnology (NY) 11: 187-193, 1993. </p>
            <p>[11] Ingham AB and Moore RJ: Recombinant production of antimicrobial peptides in heterologous microbial systems. Biotechnol Appl Biochem 47: 1-9, 2007. </p>
            <p>[12] Cabral KM, Almeida MS, Valente AP, Almeida FC and Kurtenbach E: Production of the active antifungal Pisum sativum defensin 1 (Psd1) in Pichia pastoris: overcoming the inef ciency of the STE13 protease. Protein Expres Purif 31: 115-122, 2003. </p>
            <p>[13] Feng X, Liu C, Guo J, Song X, Li J, Xu W and Li Z: Recombinant expression, purification, and antimicrobial activity of a novel hybrid antimicrobial peptide LFT33. Appl Microbiol Biotechnol 95: 1191-1198, 2012. </p>
            <p>[14] Zhou L, Zhao Z, Li B, Cai Y and Zhang S: TrxA mediating fusion expression of antimicrobial peptide CM4 from multiple joined genes in Escherichia coli. Protein Expr Purif 64: 225-230, 2009. </p>
            <p>[15] Pokorny A, Kilelee EM, Wu D and Almeida PF: The activity of the amphipathic peptide delta-lysin correlates with phospholipid acyl chain structure and bilayer elastic properties. Biophys J 95: 4748-4755, 2008. </p>
            <p>[16] Wimley WC: Describing the mechanism of antimicrobial peptide action with the interfacial activity model. ACS Chem Biol 5: 905-917, 2010. </p>
            <p>[17] Rausch JM, Marks JR, Rathinakumar R and Wimley WC: Beta-sheet pore-forming peptides selected from a rational combinatorial library: mechanism of pore formation in lipid vesicles and activity in biological membranes. Biochemistry 46: 12124-12139, 2007. </p>
          </div>

          <div class="content-block">
            <h1 id="rL">Lysis</h1>
            <div class="hr-black"></div>
            <h2>Overview</h2>
            <p><strong>Goal:</strong> To lyse E.coli stains above 42℃ and test mutant temperature-sensitive device.</p>
            <p><strong>Achievements:</strong> We built a new temperature-sensitive device and designed the mutant type of CIλrepressor. For measurement, we finished qualitative measurement and quantitative measurement of these parts and devices .Results indicate that temperature-sensitive killing device can lyse E. coil above 42℃ and</p>
            <p><strong>Parts:</strong> BBa_K1919100, BBa_K1919101, BBa_K1919102</p>

            <h2>Introduction</h2>
            <p>Background: To achieve the goal that once people wear shoes, the CecropinXJ(BBa_K1919000) can be released from cystolic, we design a temperature sensitive killing device (BBa_K1919100) .This part consists of a composite promoter (BBa_K608351)and hokD (BBa_K1497008). Composite promoter (BBa_K608351) is a part of CIλrepressor system. When the temperature is raised to 35℃, CIλrepressor start to denature and can’t bind with promoter ,then hokD gene will be expressed to kill the bacteria. At 42℃, the repressor is totally denatured. (缺模式图)</p>

            <h2>Design & Results</h2>
            <p><strong>1) Part:BBa_K1919101</strong></p>
            <p>Function: This is based on part(BBa_K608351).Composite promoter (BBa_K608351) include the mutant type CIλrepressor so that it can express above 37℃.Compared with the part(BBa_K608351),it is more sensitive at 37℃.According to the essay, we choose five important sites A44G, G92A, C188T, G378A, A629T, which may enhance more than tenth times expression at 37℃，compared with wild type(BBa_K608351).</p>
            <div class="figure">
              <img src="http://igem.org/wiki/images/a/a3/T--SCU-China--p-r-l-f11.png">
              <img src="http://igem.org/wiki/images/b/b7/T--SCU-China--p-r-l-f12.png">
            </div>
            <p>As for the measurement of this part, we choice eGFP and FCM to test the expression at 37℃.</p>
            <p>Result:  (待补充)</p>
            <p><strong>2) Device:a.BBa_K1919100(需要质粒图)</strong></p>
            <p>Introduction: Composite part consists of a composite promoter (BBa_K608351)and hokD (BBa_K1497008).Composite promoter (BBa_K608351) is a part of CIλrepressor system. When the temperature is raised to 42°C, CIλrepressor can not bind with promoter ,then hokD gene will be expressed to kill the bacteria. </p>
            <p>Qualitative measurement: After 16h incubation of E.coli with device BBa_K1919100 at 30℃, we take 1ml culture in 2ml EP tube and incubate one night .We use Methylene blue solution to identify the difference between 30℃ team and 42℃ team. The result shows that 42℃ team bacteria were almost died and dyed with Methylene blue when 30℃team bacteria survived.</p>
            <p>Quantitative measurement:1）OD600:We build two kinds of plasmids to test the expression of temperature-sensitive device (BBa_K1919100-team and BBa_K1919100 with reporter E0450-RFP team).Comparing the same E.coli strain, . Despite of the temperature factor, the OD 600 of 30℃team and 42℃team all rose before 10 hours. Then 30℃team remained the same trend until 14 hours and have a platform for 8 hours then rose again to OD600 1.6. But the 42℃team’s OD 600 showed a significant decline after that rise phase during 10 to 14 hours and then grown steadily.</p>
            <p>The OD 600 of 30℃team-RFP and 42℃team-RFP also rose before 10 hours together. Then the The OD 600 of 30℃team-RFP held this trend and rose until 14 hours then it still rose in a more gentle way. But the OD 600 of 42℃team-RFP showed a steady trend after 10 hours and grew slightly.
</p>
            <div class="figure">
              <img src="http://igem.org/wiki/images/d/d2/T--SCU-China--p-r-l-f21.png">
              <img src="http://igem.org/wiki/images/9/99/T--SCU-China--p-r-l-f22.png">
            </div>
            <p>2)Fluorescence microplate system:(差一个酶标的表)</p>
            <p>b.BBa_K1919102</p>
            <p>Natural function of part: Composite part consisting of a composite promoter (BBa_K1919101)and hokD (BBa_K1497008).Composite promoter (BBa_K1919101) include the mutant type CIλrepressor so that it will express above 37℃, so this device can kill E.coli above 37℃.</p>

          </div>

        </div>

        <div class="aqua-bk padding-bottom">
          <div class="content-block">
            <h1 class="center">Drake</h1>
            <p>Work, work, work, work, you see me I be work with “Rihanna”.
Yes, the insole is “Rihanna” featuring “Drake”. Strain “Drake” includes gene aarC to covert acetic acid into TCA cycle. It also competitively utilizes leucine to prevent the production of isovaleric acid in shoes.</p>
          </div>
        </div>
        <div class="padding-bottom">
          <div class="content-block">
            <h1 id="dOverview">Overview</h1>
            <div class="hr-black"></div>
            <p>Besides releasing AMP, we want to absorb the foot odor as well. Researches show that the isovaleric acid is key component to cause uncomfortable odor for human beings, though it only occupies 2.3% in the fatty acid components of foot odor. And another major component in the odor, acetic acid, which occupies 85.1% in the fatty acid components, contributes to the odor of sweat. (Figure 1)[1]. As a result, we decided to remove those two fatty acid components.</p>
            <div class="figure">
              <img src="http://igem.org/wiki/images/4/4c/T--SCU-China--p-d-o-f1.png">
              <p>Figure 1</p>
            </div>
            <p>In order to reduce the acetic acid production, we utilize Succinyl-Coenzyme A (CoA): Acetate CoA-Transferase (aarC), which is derived from Acidophile Acetobacter aceti. aarC can transform succinyl coenzyme A and acetic acid into acetyl coenzyme A and succinic acid.</p>
            <p>But for isovaleric acid, it’s hard to deal because we didn’t find specific enzyme to degrade it because it enters branched chain fatty acid metabolism pathway. But isovaleric acid is the degraded product of leucine which comes from the skin. As a result, we decide to competitively utilize leucine to prevent the production of isovaleric acid. (Figure 2)[2] </p>
            <div class="figure">
              <img src="http://igem.org/wiki/images/2/2d/T--SCU-China--p-d-o-f2.png">
              <p>Figure 2</p>
            </div>
            <p>We want to overexpress the ABC transport of leucine, tRNA and aminoacyl-tRNA synthetase, a special synthesized gene called polyleu, whose coding protein is full of leucine to enhance the whole protein synthesis process. But at last, we only overexpress the transport.</p>
            <p>The Drake system works as follows.</p>
            <div class="figure">
              <img src="http://igem.org/wiki/images/0/0c/T--SCU-China--i-4.gif">
              <p>Figure 3</p>
            </div>
            <p>[1] Ara, K, et al. "Foot odor due to microbial metabolism and its control. " Canadian Journal of Microbiology 52.4(2006):357-364(8).</p>
            <p>[2] James, A. Gordon, D. Cox, and Kathryn Worrall ‡. "Microbiological and biochemical origins of human foot malodour †." Flavour & Fragrance Journal 28.4(2013):231–237.</p>
          </div>

          <div class="content-block">
            <h1 id="dA">Acetic Acid</h1>
            <div class="hr-black"></div>
            <h2>Overview</h2>
            <p><strong>Goal:</strong> To reduce the acetic acid metabolized by microorganisms.</p>
            <p><strong>Achievements:</strong> The enzyme have not been successfully expressed in E. coli BL21</p>
            <p><strong>Parts:</strong> BBa_K1919233</p>

            <h2>Introduction</h2>
            <p>Acetic acid is the metabolic product of microorganisms on foot that contributes sweat odor in foot odor. In order to reduce the acetic acid production, we utilize Succinyl-Coenzyme A (CoA): Acetate CoA-Transferase (aarC), which is derived from Acidophile Acetobacter aceti. aarC can transform succinyl coenzyme A and acetic acid into acetyl coenzyme A and succinic acid. </p>
            <p>There are two benefits owing to the metabolic pathway: First, acetic acid can be involved in tricarboxylic acid cycle (TCA) rather than be discarded as waste. Second, E. coli can also use the acetic acid from outside for its growth.</p>

            <h2>Design & Result</h2>
            <p>Figure 1 shows how aarC works (Figure 1).</p>
            <p>We got the coding sequence of the enzyme in the NCBI database. After codon optimization, the gene is synthesized by Synbio Technologies Co. Ltd. We used pET32a+ vector to express the gene, and the protein expression was detected by SDS-PAGE (Figure 1).</p>
            <p>pET32a+ vector involves His-tag and T7 terminator. His-tag can be used for Western-blot to detect. We cloned aarC coding sequence, His-tag and T7 terminator from the pET32+ vector together by PCR (Figure 3).</p>
            <p>Considering the expression may bring effects on the growth of E.coli, we use four constitutive promoters at different strengths. We use long primers in to ligate standard RBS and four kinds of promoters J23100, J23112, J23113 and J23114 to the front of aarC coding sequence with His-tag and T7 terminator cloned from pET32a+ vector. (Figure 4). </p>
            <p>After promoter, RBS and aarC coding sequence have been assembled as a completed expression system, we ligate them to the pSB1C3 vector to get aarC expression vector of four kinds of expression intensity (Figure. 5).</p>
            <p>Then the expression vectors were introduced into BL21 (DE3) strains, we performed SDS-PAGE (Figure. 6), Western-blot (Figure. 7) and RT-PCR (Figure. 8) to detect the expression.</p>
            <p>To test the function of aarC, we cultured BL21(DE3) strains loaded with four types aarC expression systems in the same condition and sampled the medium in time gradient. Then the acetic acid concentration in the medium was detected by HPLC, the results obtained are in Figure 9. </p>
            <p>We also added additional acetic acid to the culture medium to test the result, and the results obtained are shown in the follow. (FIG. 10). </p>
            <p>In summary, we know that under the control of the Jxxxx promoter, the balance between energy consumption and acetic acid reduce is best. </p>
            <p>All parts have been submitted. </p>

          </div>

          <div class="content-block">
            <h1 id="dL">Leucine</h1>
            <div class="hr-black"></div>
            <h2>Overview</h2>
            <p><strong>Goal:</strong> With the aim to enhance E.coli utilize leucine dramatically, we express liv operon heterogeneously in BL21(DE3)</p>
            <p><strong>Achievements:</strong> Successfully construct 6 plasmids in which the six gene from liv operon reside flanking with the same promoter, RBS and terminator. In addition, a similar plasmid containing whole liv operon was constructed as well. The expression level of the six genes has been scrutinized via quantitative PCR. The functions of all seven devices have been tested via spectrophotometer, the results of which suggest that all of the gene can reduce the amount of leucine in LB liquid medium.</p>
            <p><strong>Parts:</strong> BBa_K1919300, BBa_K1919301, BBa_K1919302, BBa_K1919303, BBa_K1919304, BBa_K1919305, BBa_K1919306.</p>

            <h2>Introduction</h2>
            <p>Leucine is one of the element that can be metabolized to isovaleric acid by microorganisms on foot and then delivers awful odor. Thus we design a kind of special strain can utilize leucine dramatically so that other microbe can’t have this kind of raw materials to produce chemicals that have unpleasant smell. </p>

            <h2>Design</h2>
            <p>Liv operon contain six genes which are regarded as a operon coding a leucine transporter complex. All six genes used in this project derived from E.coli DH5a and in normal condition, this operon can’t start transcript because of inhibition by upstream small molecule lrp. Therefore, we clone all six genes into plasmids under control of constitutive promoter J23100. In order to make sure that all the genes in the plasmids are expressed, we utilize quantitative PCR to scrutinize the expression level of all six gene in the operon respectively in both BL21(DE3) and DH5a. Although it’s well known that BL21(DE3) is quite suitable for gene expression, we considered that the genes are derived from DH5a and thus it’s necessary to observe the expression level of genes in BL21(DE3) compared with those in DH5a. What’s more, with the aim to promote microbe to absorb leu in medium further, we attempt to clone tRNA-leu, aminoacyl tRNA synthetase and artificially synthesized polyleucine fragment which contain about 30 tandem leucines into overexpression vectors accordingly. Unfortunately, all those vectors are failed to be constructed, except for polyleucine fragment. Nevertheless, as figure A shows, the secondary structure of this gene’s transcript is too complex and are believed can not transcript properly in vivo. Thus, the polyleucine was discarded later in the project. The results of qPCR show that the expression levels of genes are accelerated when the E.coli was introduced into the plasmids, which means all six genes have been overexpressed in the microbe.
</p>
            <div class="figure">
              <img src="http://igem.org/wiki/images/a/a1/T--SCU-China--p-d-l-fa1.png">
              <p>figureA1  polyleucine RNA secondary structure prediction based on MFE(minimum free energy). The color indicates base-pair probabilities. </p>
              <img src="http://igem.org/wiki/images/d/d0/T--SCU-China--p-d-l-fa21.png">
              <img src="http://igem.org/wiki/images/5/50/T--SCU-China--p-d-l-fa22.png">
              <p>FigureA2  Sequence of polyleucine fragment</p>
              <img src="http://igem.org/wiki/images/7/7d/T--SCU-China--p-d-l-fa3.png">
              <p>Figure A3 entropy of polyleucine fragment</p>
              <img src="http://igem.org/wiki/images/3/39/T--SCU-China--p-d-l-fa4.png">
              <p>Figure A4  the dot plot containing the base pair probabilities of polyleucine fragment.</p>
              <img src="http://igem.org/wiki/images/2/21/T--SCU-China--p-d-l-fa5.png">
              <p>FigureA5</p>
            </div>
            <p>Since the polyleucine fragment is artificially synthesized by company and they do not make it completely, we repaired the fragment by adding promoter and RBS via Gibson assembly. The figure shows different condition to amplify the fragment of polyleucine. As mentioned before, there are numerous short repeats in the fragment which can account for why there are ladder like bands in the figure. </p>
            <p>In addition, we noted that the expression level of livJ is increased drastically compared with other gene in the operon, which may indicate that the livJ plays a crucial role in metabolism of E.coli, which is consistent with the references that livJ is the most sensitive gene to lrp regulation in the operon. What’s more, we interestingly find that all expression levels of six genes are reduced generally in the same sequence of their position in operon, which might shed light on the fact that the stability of RNA of those genes in the operon is determined by coding sequence. Definitely, we can’t neglect the role of UTR (untranslated region) of liv operon played in regulation of stability of RNA in wild type of E.coli. However, as we have mentioned before, all these genes’ coding sequence starting from ATG to TAA are cloned to the overexpression vectors, in another word, the only discrepancy of overexpression vectors is the coding sequence and 5’ UTR & 3’UTR are all the same because of identical promoter, RBS and terminator. What’s more, not only the BL21(DE3) shows this kind of tendency that the expression level of six gene decrease in order, but the DH5a shows the tendency as well. Thus, we have sufficient evidence to support our belief that the determinant of stability of gene’s RNA includes in ORF (open reading frame).</p>
            <p>After ensuring that the gene can be overexpressed in E.coli, we go on to examine the function of these genes. Although there are service in company to test the amount of amino acid in liquid sample, we incline to finish our project by ourselves as much as possible. We choose spectrophotometer to observe function of genes. In order to ensure the amount of leucine can be detected by spectrophotometer, we artificially increase the concentration of leucine in liquid LB to about 1 g/L. Thus, the spectrophotometer can detect the alternation of concentration of leucine after incubation for more than 24 hours with engineered E. coli. </p>
            <p>First of all, we established a standard method to dispose the LB medium in this project so that all the data can be comparable. The results of spectrophotometer indicate that all six genes can deduce the concentration of leucine in liquid medium, albeit in different level. However, the results are not very convincing since the background of amino acid in LB medium can disturb the measurement profoundly. Then, we try to utilize the inorganic medium, M9 medium, to inoculate the engineered E.coli and the medium is artificially laced with additionally leucine. The results suggest that our engineered microbe do own desired function, that is, absorb the leucine from ambiance.</p>

            <h2>Result</h2>
            <p>1. Gibson Assembly</p>
            <div class="figure">
              <img src="http://igem.org/wiki/images/b/b5/T--SCU-China--p-d-l-f1a.png">
              <p>Figure1A</p>
              <img src="http://igem.org/wiki/images/d/de/T--SCU-China--p-d-l-f1b.png">
              <p>Figure 1B</p>
              <img src="http://igem.org/wiki/images/3/30/T--SCU-China--p-d-l-f1c.png">
              <p>Figure 1C</p>
            </div>
            <p>As the figure shows, we construct our overexpression vectors via Gibson assembly. However, because some of unknown problem occur in the reaction buffer, we did not get clear bands. It may result from long time storage of the assembly mix and the activity of three different enzyme in the mix altered, the activity of exonuclease may preponderate over other enzyme and digest the fragment into piece. Fortunately, we have screen out the positive colony after transformation.</p>
            <div class="figure">
              <img src="http://igem.org/wiki/images/2/2c/T--SCU-China--p-d-l-f2a.png">
              <p>Figure 2A</p>
              <img src="http://igem.org/wiki/images/1/1a/T--SCU-China--p-d-l-f2b.png">
              <p>Figure 2B The six gene of the liv operon and the position of the six genes</p>
              <img src="http://igem.org/wiki/images/b/b9/T--SCU-China--p-d-l-f3a.png">
              <p>Figure 3A</p>
              <img src="http://igem.org/wiki/images/4/49/T--SCU-China--p-d-l-f3.5a.png">
              <p>The result of relatively quantitative PCR. We utilize two gene as reference in the PCR, one is rrsA which is a gene of 16s RNA and the other is dnaA which is a gene of DNA polymerase element. Generally speaking, the expression level of rrsA is very high, whereas the expression level of dnaA is relatively low. In this table, we presume that the discrepancy of dnaA’s and rrsA’s expression level are the same in BL21 and DH5a, which is 10^10 folds. </p>
              <img src="http://igem.org/wiki/images/a/ad/T--SCU-China--p-d-l-f3b.png">
              <p>Figure3B</p>
              <img src="http://igem.org/wiki/images/8/80/T--SCU-China--p-d-l-f3c.png">
              <p>Figure3C</p>
            </div>
            <p>However, we found the discrepency of expression level of dnaA and rrsA are quite distinct between BL21(DE3) and DH5a. T-test indicates p&lt;0.01 (table 1B). Thus, we take this factor that the expression level of rrsA is 100000 folds over expression level of dnaA in DH5a and 160000 folds in BL21(DE3) into account.</p>
            <div class="figure">
              <img src="http://igem.org/wiki/images/d/d2/T--SCU-China--p-d-l-f3d.png">
              <p>figure 3D</p>
              <img src="http://igem.org/wiki/images/8/8d/T--SCU-China--p-d-l-f3e.png">
              <p>figure 3E</p>
            </div>
            <p>After take the factor mentioned before into account, we get the figure 3D and 3E which reveal more reality than figure 3A does.</p>
          </div>
        </div>

        <div class="aqua-bk padding-bottom">
          <div class="content-block">
            <h1 class="center">Enhancement</h1>
            <p>E. coli odor itself is not that pleasant as well. We use RED to knockout the insole production gene tnaA (tryptophanase) to reduce the odor itself. Insole in shoes is not a wonderful home for microorganisms to grow. We use vitreoscilla hemoglobin (existing part K1321200) to enhance the grow ability of two E. coli strains and get the quantitative measurement result of the Biobrick.</p>
          </div>
        </div>
        <div class="padding-bottom">
          <div class="content-block">
            <h1 id="eOverview">Overview</h1>
            <div class="hr-black"></div>
            <p>Apart from the foot odor, the smell of E. coli itself is also a problem remained to be solved, which is imperative for gaining acceptability for our product. The smell is believed that mainly comes from the derivatives of indole which mostly originate from indole. Thus shutting down the production of indole becomes the key task. The direct way of indole formation in E. coli is the process that turns a tryptophan into indole by tryptophanase. As a result, turning off this process by knocking out tryptophanase will reduce the source of indole as well as the smell. We decide to use RED system to knockout the gene coding tryptophanase: tnaA. [1]</p>
            <p>Insole in shoes is not a wonderful home for microorganisms to grow. The oxygen and nutrition are both limited. We use vitreoscilla hemoglobin (existing part K1321200) to enhance the grow ability of two E. coli strains and get the quantitative measurement result of the Biobrick. And it’s the “Biobrick character improvement” in our project this year. We prove that it can be expressed in E.coli and get a lot of quantitative measurement result.</p>
            <p>[1] 2014 iGEM Team:Paris_Saclay, “Remove the bad smell of E.coli” <a href="http://2014.igem.org/Team:Paris_Saclay/Project/Odor-free_ecoli">http://2014.igem.org/Team:Paris_Saclay/Project/Odor-free_ecoli</a>， 2016/10/4</p>
          </div>

          <div class="content-block">
            <h1 id="eK">Knockout</h1>
            <div class="hr-black"></div>
            <p class="extra-margin-bottom">Foot odor and tinea pedis (athlete's foot) are such common foot diseases. Foot odor can always cause embarrassing trouble in public areas. And tinea pedis (athlete's foot) makes patient’s foot itching, scaling, and redness. A lot of people in Sichuan Province of China suffer from this, which is consistent with our human practice survey result.</p>
          </div>
        </div>
      </div><!-- /.container -->
  </body>
</html>
